slices/binutils-aarch64-linux-gnu.yaml

```
@@ -0,0 +1,87 @@
+package: binutils-aarch64-linux-gnu
+
+essential:
+  - binutils-aarch64-linux-gnu_copyright
+
+slices:
+  # NOTE: You might want to create a symlink from /usr/bin/as to
+  #       /usr/bin/aarch64-linux-gnu-as in your rootfs to use these slices.
+  #       ln -s "/usr/bin/${arch}-as" "${rootfs}/usr/bin/as"
```

```suggestion
  #       ln -s "${arch}-as" "${rootfs}/usr/bin/as"
```

Should this be a relative link?